proxy/fuse/fuse_darwin.go

```diff
@@ -0,0 +1,28 @@
+package fuse
+
+import (
+	"os"
+
+	"github.com/GoogleCloudPlatform/cloudsql-proxy/logging"
+)
+
+const (
+	macfusePath = "/Library/Filesystems/macfuse.fs/Contents/Resources/mount_macfuse"
```

Comment:
This comes from https://github.com/hanwen/go-fuse/blob/0f728ba15b38579efefc3dc47821882ca18ffea7/fuse/mount_darwin.go#L121-L124.

Reply:
Can we make this a comment in the code as well?